src/client/src/translations/en.json

```diff
@@ -140,7 +140,6 @@
   "selectPages.pagesTitleQuestion": "Select pages for your application.",
   "selectPages.webAppTitleQuestion": "Select a project type.",
   "selectableCard.details": "Details",
-  "selectableCard.selected": "Selected",
```

Comment:
Is it not being used now? Did you run `npm run manage:translations`?

Reply:
yeah I deleted the text

Reply:
since it's not being used anywhere now